src/components/Square.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { updateBoardState } from 'utils'
+
+//Square props
+type SquareProps = {
+  id: number
+  disabled?: boolean
+  isHighlighted: boolean
+  gameState: GameState
+  dispatch: React.Dispatch<Action>
+}
+
+//component
```

Comment:
no need for these comments (this one and the type comment above), I'd use comment for mostly for complex business logic. No need for comments for comments sake